cosa/build.go

```diff
@@ -44,5 +46,5 @@ type Build struct {
 			UncompressedSHA256 string `json:"uncompressed-sha256"`
 		} `json:"openstack"`
 	} `json:"images"`
-	OSTreeVersion string `json:"ostree-version"`
+	FedoraCoreOSParentVersion string `json:"fedora-coreos.parent-version"`
```

Comment:
Is the original `OSTreeVersion` used in RHCOS? If so might make sense to put both w/ a `,omitempty`

Reply:
`omitempty` is only actually used when serializing back the object to JSON, right? We don't do this today, but if we ever do, we'll probably want to match as much as possible the cosa semantics, which will probably involve slapping `omitempty` a bit everywhere. :)